modules/ROOT/pages/rate-limiting-sla-policy.adoc

```diff
@@ -193,7 +193,7 @@ Each response header has information about the current state of the request:
 * X-Ratelimit-Limit: The maximum available requests per window 
 * X-Ratelimit-Reset: The remaining time, in milliseconds, until a new window starts
 
-MuleSoft advises that you expose these headers only if the API is used within the organization. In a public API scenario, this data can be used maliciously to invoke spikes in traffic that leave your API without any quota left.
+By default, the X-RateLimit headers are disabled in the response. You can enable these headers by selecting `Expose Headers` when you configure the policy.
```

Comment:
Is "Expose Headers" in the UI (to be bold font `*Expose Headers*`) or a code-like with back ticks, as shown above?

Reply:
Done! Thanks!